pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableImplV4.java

```diff
@@ -48,7 +50,26 @@ public DataTableImplV4(int numRows, DataSchema dataSchema, Map<String, Map<Integ
   }
 
   @Override
-  protected int getDataBlockVersionType() {
+  public RoaringBitmap getNullRowIds(int colId) {
+    // _fixedSizeData stores two ints per col's null bitmap: offset, and length.
```

Comment:
Move the implementation into the `BaseDataBlock`

Reply:
This method depends on `_rowSizeInBytes` which is only defined in `RowDataBlock`. Will move to `RowDataBlock`.